text/0000-deprecate-windows-on-x86_64-non-WSL-support.md

```diff
@@ -0,0 +1,148 @@
+
+
+(all times in minutes, sourced f rom [this CI](https://github.com/ember-cli/ember-cli/actions/runs/9862588536))
+
+```
+# (1) => (10.5 + 32)
+#        (longest times for each stage of CI) 
+#        42.5
+# (2) => (1) * 8 
+#        340      # initial amount of time all PRs take in CI
+# (3) => (1) * 7 
+#        297.5      # minimal number of re-runs, for manual updating of a PR 
+# (4) => (3) + (2)
+#        637.5      # minimal amount of CI time, if no flakey tests run
+```
+
+
+And without windows, this would be: 
+```
+# (1) => (1.9 + 12)
+#        13.9
+# (2) => (1) * 8 
+#        111.2      # initial amount of time all PRs take in CI
+# (3) => (1) * 7 
+#        97.3      # minimal number of re-runs, for manual updating of a PR 
+# (4) => (3) + (2)
+#        208.5      # minimal amount of CI time, if no flakey tests run
+```
+
+Total improvement: 3.06xx (without windows takes 33% of the time)
+
+
+
+> Why are we doing this? What use cases does it support? What is the expected
+outcome?
+
+## Detailed design
+
+This is a WIP RFC, and I'm more curious about what the community thinks if this is a bad idea, and if official windows support should be kept or not.
+
+> This is the bulk of the RFC.
+
+> Explain the design in enough detail for somebody
+familiar with the framework to understand, and for somebody familiar with the
+implementation to implement. This should get into specifics and corner-cases,
+and include examples of how the feature is used. Any new terminology should be
+defined here. 
+
+> Please keep in mind any implications within the Ember ecosystem, such as:
+> - Lint rules (ember-template-lint, eslint-plugin-ember) that should be added, modified or removed
+> - Features that are replaced or made obsolete by this feature and should eventually be deprecated
+> - Ember Inspector and debuggability
+> - Server-side Rendering
+> - Ember Engines
+> - The Addon Ecosystem
+> - IDE Support
+> - Blueprints that should be added or modified
+
+## How we teach this
+
+> What names and terminology work best for these concepts and why? How is this
+idea best presented? As a continuation of existing Ember patterns, or as a
+wholly new one?
+
+> Would the acceptance of this proposal mean the Ember guides must be
+re-organized or altered? Does it change how Ember is taught to new users
+at any level?
+
+> How should this feature be introduced and taught to existing Ember
+users?
+
+> Keep in mind the variety of learning materials: API docs, guides, blog posts, tutorials, etc.
+
+## Drawbacks
+
+> Why should we *not* do this? Please consider the impact on teaching Ember,
+on the integration of this feature with other existing and planned features,
+on the impact of the API churn on existing apps, etc.
+
+> There are tradeoffs to choosing any path, please attempt to identify them here.
+
+## Alternatives
```

Comment:
Maybe refactoring the ember cli test suite to not take as long should go here?

Reply:
absolutely -- but this problem extends beyond ember-cli, too -- all CIs in all projects that test windows

Reply:
another alternative:
running tests in parallel: https://github.com/ember-cli/ember-cli/pull/10488